clients.json

```
@@ -628,6 +628,15 @@
     "active": true
   },
 
+  {
+    "name": "LettuceF",
+    "language": "Scala",
+    "repository": "https://github.com/naoh87/lettucef",
+    "description": "Scala FP wrapper for Lettuce with Cats Effect",
+    "authors": ["naoh"],
```

This should be your Twitter handle - please confirm you are indeed https://twitter.com/noah, or just leave this empty.

```suggestion
    "authors": [],
```

---

Thank you very much for quick reply.
Now I see what I should write.
I updated twitter handle and profiles, and also GitHub profile.